spec/controllers/users_controller_spec.rb

```diff
@@ -23,15 +23,19 @@ def valid_request
         post :create, user: FactoryGirl.attributes_for(:user)
       end
 
+      it "user should be assigned one organization_id" do
+          user.organization = organization
+          user.save
+          expect(user.organization_id).to be(organization.id)
+      end
```

Comment:
This test doesn't make sense and it will always be true. You're testing the controller so you should send the a request to the controller at least.